.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+name: Test the packages
+on:
+  push:
+    branches:
+      - "master"
+      - "development"
+    paths-ignore:
+      - "**/README.md"
+      - "docs/**"
+  pull_request:
+    branches:
+      - "**"
+
+jobs:
+  test-dart:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        sdk: [stable, beta, dev]
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v3
+
+      - name: Setup dart
+        uses: dart-lang/setup-dart@v1
+        with:
+          sdk: ${{ matrix.sdk }}
+
+      - name: Install dependencies
+        run: dart pub get --directory packages/dart
+
+      - name: Run build_runner
+        run: (cd packages/dart && dart run build_runner build --delete-conflicting-outputs)
+
+      - name: Verify formatting
+        run: dart format --output=none --set-exit-if-changed packages/dart
+
+      - name: Analyze project source
+        run: dart analyze packages/dart
+
+      - name: Run tests
+        run: (cd packages/dart && dart test)
+
+  test-flutter:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        channel: [stable, beta]
```

Comment:
Why is there no `dev` channel but there is for dart?

Reply:
![grafik](https://user-images.githubusercontent.com/45403027/166084513-c18c29cd-865f-4b07-a8e6-5fd92b478456.png)
On the top, it has not been listed as a valid option, but as I've noticed now, it seems to be (bellow). I'll add it.